### PR TITLE
Adding issue template

### PR DESCRIPTION
**Pull request by jwallra**

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+_Please list the steps used to reproduce your issue._
+
+1.
+2.
+
+### **Details**
+
+1. Which app platform you are developing for?
+    - _Android/Apple/iOS_
+2. Which OS version did you experience the issue on?
```
Comment (elamalani) [inline issue_template.md]: What about SDK version that you are using for App Center?

File: issue_template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    - _e.g. iOS 11, Android 8.1.0_
+3. What device version did you see this error on?  Were you using an emulator or a physical device?
+    - _e.g. iPhone X physical device, Google Pixel 2 emulator_
+4. What third party libraries are you using? <!-- For .NET, you can find these in your .csproj, package.json, or package.config files -->
+    - _example_
+5. Please enable verbose logging for your app using the line below and include the logs here.
```
Comment (elamalani) [inline issue_template.md]: We should say that enable verbose logging by using the API below right before AppCenter.start() API

File: issue_template.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+### **Repro Steps**
+
+_Please list the steps used to reproduce your issue._
+
+1.
+2.
+
+### **Details**
+
+1. Which .NET SDKs are you using?
```
Comment (elamalani) [inline issue_template.md]: Can we combine 1 and 2 to say - what is your app OS and platform?
[ ] UWP
[ ] Xamarin.Android
[ ] Xamarin.iOS
[ ] Xamarin.Forms

File: issue_template.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    - [ ] UWP
+    - [ ] Xamarin.Android
+    - [ ] Xamarin.iOS
+    - [ ] Xamarin.Forms
+2. Which app platform you are developing for?
+    - _Android/Apple/iOS_
+3. Which OS version did you experience the issue on?
```
Comment (elamalani) [inline issue_template.md]: We should ask about SDK version as well.

**Comment by jwallra** [comment]
@elamalani let me know if you have any more comments or go ahead and merge it if it looks good

**Pull request merged**